tutorials/katacoda/thanos/2-lts/step3.md

```diff
@@ -0,0 +1,65 @@
+# Step 3 - Installing the Thanos Store
+
+In this step, we will learn about Thanos Store Gateway, how to start, and what problems are solved by it.
+
+## Thanos Components
+
+Thanos is a single Go binary capable to run in different modes. Each mode represents a different component and can be invoked in a single command.
+
+Let's take a look at all the Thanos commands:
+
+```docker run --rm quay.io/thanos/thanos:v0.13.0 --help```{{execute}}
+
+You should see multiple commands that solve different purposes, block storage based long-term storage for Prometheus.
+
+In this step we will focus on thanos `store gateway`:
+
+```
+  store [<flags>]
+    Store node giving access to blocks in a bucket provider
+```
+
+## Store Gateway/ Store :
+
+* This component implements the Store API on top of historical data in an object storage bucket. It acts primarily as an API gateway and therefore does not need significant amounts of local disk space.
+* It joins a Thanos cluster on startup and advertises the data it can access.
+* It keeps a small amount of information about all remote blocks on the local disk and keeps it in sync with the bucket.
+This data is generally safe to delete across restarts at the cost of increased startup times.
+
+
+You can read more about [Store](https://thanos.io/components/store.md/) here.
+
+## Deployment
+
+Before deploying, we need to stop the running thanos `Sidecar` here, we can do so by running:
+
+```
+docker stop sidecar
+```{{execute}}
+
+Now, click snippets to deploy thanos store to the running Prometheus instance.
+
+### Deploying store to "EU1" Prometheus
+
+```
+docker run -d --net=host --rm \
+    -v $(pwd)/bucket_storage.yml:/etc/prometheus/bucket_storage.yml \
+    -v $(pwd)/test:/prometheus \
+    --name thanos-store \
+    quay.io/thanos/thanos:v0.13.0 \
+    store \
+    --data-dir             /prometheus \
+    --objstore.config-file /etc/prometheus/bucket_storage.yml \
+    --http-address         0.0.0.0:19190 \
+    --grpc-address         0.0.0.0:19091
+```{{execute}}
+
```

Comment:
I am missing step which will allow users to actually query Thanos store data

Reply:
Yes, adding it and I am also including an image as an example to show how it will look. Kindly check it, and we can see if it looks cool here or not 👍